webrtc.html

```diff
@@ -9419,10 +9417,17 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                     <li>Remove the first character from <code><a data-for=
                     "RTCDTMFSender">toneBuffer</a></code> and let that
                     character be <var>tone</var>.</li>
-                    <li>Start playout of <var>tone</var> for <code><a data-for=
+                    <li>If <var>tone</var> is "," send silence for
```

Comment:
Should say "delay sending tones" instead of "send silence", since if you're sending audio you don't want to actually mute audio for 2 seconds. This should just insert a 2-second gap in between tones.